### ISSUE TITLE
Missing keyboard shortcut to “Run task…” command

### ISSUE THREAD

**Issue by pakLebah** [microsoft/vscode#108051]
System info:
- VSCode Version: 1.49.3 (also with 1.49.2)
- OS Version: MacOS Mojave 10.14.6

Steps to reproduce:
1. Open _Preference_ → _Keyboard Shortcuts_.
2. Search for “run task” will shows none.

Does this issue occur when all extensions are disabled? **Yes**

Expected outcome:
_Terminal | Run Task…_ command used to have ⌘⇧R shortcut (AFAIR) or get custom keyboard shortcut.

Screenshot:
![Jepretan Layar 2020-10-04 pukul 07 53 34](https://user-images.githubusercontent.com/11470932/95004610-d0add600-0617-11eb-9264-f221e89f1b9b.png)

**Comment by pakLebah** [comment]
It's strange, when I opened VS Code on Windows, the command is available and assignable.

![Cuplikan layar 2020-10-04 092601](https://user-images.githubusercontent.com/11470932/95005558-5afc3700-0624-11eb-90c3-1dfe84a146e7.png)

**Comment by alexr00** [comment]
I don't think we've ever defined a default keyboard shortcut for Run Task. Can you open your custom keyboard shortcuts and see if you define it?

**Comment by pakLebah** [comment]
@alexr00: It’s not about whether it has default keyboard or not. The Run Task command is not available on Mac while it’s available on Windows. See the image in the first post.

**Comment by alexr00** [comment]
Turns out that we don't show commands that take args in the keybindings editor: https://github.com/microsoft/vscode/issues/84072

You're seeing it on Windows but not Mac because you have your own custom keyboard shortcut setup on Windows.

**Comment by pakLebah** [comment]
I fresh and clean installed Mojave on my Mac. It has nothing configured by me, all are still in default. I don't even bother to login into iCloud. Then I installed VS Code on it, the first and only app I installed on the fresh and clean MacOS. Then I opened the Keyboard Shortcuts setting... and the Run Task command is still **_not_** there. It's exactly like my first image above. Is it by design too? Did you mean MacOS don't get the Run Task command in the Keyboard Shortcuts is by design? C'mon.

**Comment by pakLebah** [comment]
Out of curiosity, I removed the latest VS Code I just installed. Then I installed VS Code v.1.43.2 from February 2020, the version that I remembered having the Run Task command. And... it does have the command. Here's the screenshot.

![Jepretan Layar 2020-10-09 pukul 20 46 12](https://user-images.githubusercontent.com/11470932/95590977-efc1c300-0a70-11eb-9caf-b2403229f847.png)

So, the keyboard shortcuts setting has a redesigned since v.1.45?
